Assets/Tests/InputSystem/CoreTests_Layouts.cs

```diff
@@ -896,6 +896,78 @@ public void Layouts_CanApplyOverridesToExistingLayouts()
         Assert.That(device["extraControl"], Is.TypeOf<ButtonControl>());
     }
 
+    [Test]
+    [Category("Layouts")]
+    public void Layouts_CanReplaceExistingOverrideToExistingLayouts()
+    {
+        // Add a control to existing Mouse layout.
+        InputSystem.RegisterLayoutOverride(@"
+            {
+                ""name"" : ""Overrides"",
+                ""extend"" : ""Mouse"",
+                ""controls"" : [
+                    { ""name"" : ""extraControl"", ""layout"" : ""Button"" }
+                ]
+            }
+        ");
+
+        //var device = InputSystem.AddDevice<Mouse>();
+
+        //Assert.That(device["extraControl"], Is.TypeOf<ButtonControl>());
```

Comment:
No commented out code pls. (except in exceptional cases and with comment)

Reply:
Removed, mistake